test/Dockerfile.test

```diff
@@ -28,11 +29,21 @@ RUN ln -fs /usr/share/zoneinfo/America/New_York /etc/localtime && \
     dpkg-reconfigure --frontend noninteractive tzdata && \
     apt-get install -y postgresql
 
-COPY --from=GOLANG /usr/local/go /usr/local/go
+RUN curl -L https://dl.google.com/go/go${GO_VERSION}.linux-amd64.tar.gz \
+	-o /tmp/go.tar.gz && \
+	cd /tmp && tar -xf go.tar.gz
 
-ENV KONG_GO_PLUGINSERVER_VERSION=master
 ENV GOPATH=/tmp/go
-ENV PATH $GOPATH/bin:/usr/local/go/bin:$PATH
+ENV PATH $GOPATH/bin:$PATH
+
+RUN go version ; \
+	mkdir /gps ; cd /gps ; \
+	go mod init go-pluginserver ; \
+	go get -d -v github.com/Kong/go-pluginserver@${KONG_GO_PLUGINSERVER_VERSION} ; \
+	go install -ldflags="-s -w -X main.version=${KONG_GO_PLUGINSERVER_VERSION}" ... ; \
+	cp /tmp/go/bin/go-pluginserver /usr/local/bin/ ;\
+	cd ; rm -r /gps; \
+	go-pluginserver --version

```

Comment:
if we also removed the go tests do we require golang and go-pluginserver?

Reply:
This Dockerfile is used for running Kong tests, which contains an integration test for Go plugins.

Reply:
right good point. Are we going to eventually migrate this to `curl the go-pluginserver` release from github or in both testing and using `go-pluginserver` is a custom Kong docker image the route to proceed with?

Reply:
@hutchic, as for the first question, we could do that, since we already tag releases and have the sources available in GH releases. But what we are doing here has the same effect of fetching the tag. Does that answer?